Database.py

```diff
@@ -11,7 +11,7 @@ def __connect(cls):
         :return: none
         """
         if cls.__connection is None:
-            cls.__connection = pymssql.connect(server='cisdbss.pcc.edu', user='275student', password='275student',
+            cls.__connection = pymssql.connect(server='cisdbss.pcc.edu', user='', password='',
```

Comment:
Thank you for removing the username and password. Overall we don't want sensitive information in our public repositories.